utils/symbolicate-linux-fatal

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python
+# symbolicate-linux-fatal - Symbolicate Linux stack traces -*- python -*-
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See http://swift.org/LICENSE.txt for license information
+# See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+# ----------------------------------------------------------------------------
+#
+# Symbolicates fatalError stack traces on Linux. Takes the main binary
+# and a log file containing a stack trace. Non-stacktrace lines are output
+# unmodified. Stack trace elements are analysed using reconstructed debug
+# target matching the original process in where shared libs where mapped.
+#
+# TODOs:
+# * verbose output
+# * search symbols by name for the not <unavailable> ones
+#
+# ----------------------------------------------------------------------------
+
+from __future__ import print_function
+
+import argparse
+import sys
+import lldb
+import subprocess
+
+DYN_LIBS = {}
```

Comment:
As with `BINARY`, I think this program would be easier to understand if `DYN_LIBS` was instantiated in `main()`, then passed in as an argument to the `process_ldd()` and `process_stack()` functions.